src/open_clip/model.py

```diff
@@ -248,6 +248,47 @@ def forward(self, image, text):
         return image_features, text_features, self.logit_scale.exp()
 
 
+
+class TextTextCLIP(nn.Module):
```

Comment:
this is completely duplicated from above class, I wonder if we could reconcile it

Reply:
Maybe we could use this one as the general model? It's not specific about modality. We can refer to `image_features` as `features_a` and `text_features` as `features_b`.